src/main/java/org/truffleruby/interop/InteropNodes.java

```diff
@@ -557,14 +553,10 @@ public Object toNative(
             }
         }
 
-        protected Node createToNativeNode() {
-            return Message.TO_NATIVE.createNode();
-        }
-
     }
 
     @CoreMethod(names = "read", isModuleFunction = true, required = 2)
-    @ImportStatic({ StringCachingGuards.class, StringOperations.class })
+    @ImportStatic({ StringCachingGuards.class, StringOperations.class, Message.class })
     public abstract static class ReadNode extends CoreMethodArrayArgumentsNode {
 
         @Specialization
```

Comment:
`createReadNode()` below could be inlined in the `Cached` expression.

Reply:
Fixed.